http/headers/content-disposition.json

```diff
@@ -15,7 +15,11 @@
               "version_added": "12"
             },
             "firefox": {
-              "version_added": true
+              "version_added": true,
+              "notes": [
+                "From version 82 the inline directive is ignored if the HTML5 'a download' attribute is specified (to match the specification)",
```

Comment:
A suggestion and a question here.

Suggestion: I think these two notes tell the same story, so we could probably combine them. Also, I had a little difficulty following the `a download` story—I had to read the bug to figure out. What do you think of this rewrite that combines both?

```
"notes": "From version 82, if an <code>&lt;a&gt;</code> element's <code>download</code> attribute is set then the <code>inline</code> directive is ignored. Earlier versions did not match the specification and respected the header directive over the attribute. See <a href='https://bugzil.la/1658877'>bug 1658877</a>."
```

Question: Do you know if this applies to Firefox for Android as well? (If so, I'd copy whatever final note we get to the "firefox_android" entry below.)